api/src/main/java/org/eclipse/microprofile/reactive/messaging/Acknowledgment.java

```diff
@@ -37,6 +47,280 @@
+ * </div>
+ * </div></div></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>Manual</p></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>None, Pre-Processing, Post-Processing (when
+ * the <code>onNext</code> method returns), Manual</p></td>
+ * </tr>
+ * <tr>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><div class="content"><div class="listingblock">
+ * <div class="content">
+ * <pre><code>@Incoming("channel")
+ * SubscriberBuilder&lt;I, <span style="color:#0a8;font-weight:bold">Void</span>&gt; method()</code></pre>
+ * </div>
+ * </div></div></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>Post-Processing</p></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>None, Pre-Processing, Post-Processing (when
+ * the <code>onNext</code> method returns)</p></td>
+ * </tr>
+ * <tr>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><div class="content"><div class="listingblock">
+ * <div class="content">
+ * <pre><code>@Incoming("channel")
+ * <span style="color:#339;font-weight:bold">void</span> method(I payload)</code></pre>
+ * </div>
+ * </div></div></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>Post-Processing</p></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>None, Pre-Processing, Post-Processing (when
+ * the method returns)</p></td>
+ * </tr>
+ * <tr>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><div class="content"><div class="listingblock">
+ * <div class="content">
+ * <pre><code>@Incoming("channel")
+ * CompletionStage&lt;?&gt; method(Message&lt;I&gt; msg)</code></pre>
+ * </div>
+ * </div></div></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>Manual</p></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>None, Pre-Processing, Post-Processing (when
+ * the returned <code>CompletionStage</code> is completed), Manual</p></td>
+ * </tr>
+ * <tr>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><div class="content"><div class="listingblock">
+ * <div class="content">
+ * <pre><code>@Incoming("channel")
+ * CompletionStage&lt;?&gt; method(I payload)</code></pre>
+ * </div>
+ * </div></div></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>Post-Processing</p></td>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><p>None, Pre-Processing, Post-Processing (when
+ * the returned <code>CompletionStage</code> is completed)</p></td>
+ * </tr>
+ * <tr>
+ * <td style="border: 1px solid #999; padding: 0.5rem; text-align: left;"><div class="content"><div class="listingblock">
+ * <div class="content">
+ * <pre><code>@Incoming("in")
+ * <span style="color:#007">@Outgoing"out")
```

Comment:
@Outgoing"out") =>
@Outgoing("out")

You need to do a global replacement as there are many occurrence.

Reply:
Damn, I fixed it for @Incoming, but not for @Outgoing.

Reply:
Fixed.